Pakrun/settings.py

```diff
@@ -165,4 +165,4 @@
 
 CSRF_HEADER_NAME = 'HTTP_X_CSRFTOKEN'
 
-CSRF_TRUSTED_ORIGINS = ["https://event-management-q04c.onrender.com"]
\ No newline at end of file
+CSRF_TRUSTED_ORIGINS = ["https://event-management-q04c.onrender.com"]

```

Comment:
i had deployed the app on my own  render acc , thats why i had changed this the new app is [here](https://event-management-q04c.onrender.com/login/?next=/), however i had challenges with creating a superuser, how did you do it ?check the build.sh to see what i had done , i dont know what i did wrong

Reply:
check the build.sh that i have created